src/access/acp.test.ts

```diff
@@ -2618,3 +2619,569 @@ describe("getActorAccess", () => {
     });
   });
 });
+
+describe("internal_getActorAccessAll", () => {
+  it.each([acp.agent, acp.group])(
```

Comment:
Given that this hits the same code path and the code doesn't (nor shouldn't it) distinguish between the different types of actors, as far as I'm concerned this isn't necessary :) (Can't harm, but just in case it's going to cause you more work in finishing this up.)

Reply:
Yeah I only realized after I changed it that it didn't cover anything specific. No harm in leaving it in I guess.